src/cmd/server/start.ts

```diff
@@ -1,5 +1,6 @@
 import logger from '@pnpm/logger'
 import {createServer} from '@pnpm/server'
+import Diable = require('diable')
```

Comment:
I think you can `import Diable from 'diable'` as well.

Reply:
no, it is undefined when I import it via import from.